lib/python/rose/config_editor/util.py

```diff
@@ -201,12 +194,12 @@ def wrap_string(text, maxlen=72, indent0=0, maxlines=4, sep=","):
     return "\n".join(lines)
 
 
-def null_cmp(x, y):
-    """Compares sort_key and then id of the tuple x and y to be sorted."""
-    x_sort_key = x[0]
-    x_id = x[1]
-    y_sort_key = y[0]
-    y_id = y[1]
+def null_cmp(x_item, y_item):
+    """Compares sort_key and then id of the tuples x_item/y_item."""
+    x_sort_key = x_item[0]
+    x_id = x_item[1]
+    y_sort_key = y_item[0]
+    y_id = y_item[1]
```

Comment:
What about ``x_sort_key, x_id = x_item[:2]`` (and then the y equivalent) to halve the line count for 199-202? Or is this considered bad style/practice?